src/main/java/org/example/ngevaticketmanagerspring/user/UserService.java

```diff
@@ -0,0 +1,63 @@
+package org.example.ngevaticketmanagerspring.user;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    private final UserRepository repository;
+
+    private final String exceptiontext = "Could not find user with ID: ";
```

Comment:
```suggestion
    private final String EXCEPTION_TEXT = "Could not find user with ID: ";
```
Weil Konstante. Oder sonst "exceptionText".